plutus-core-spec/figures/PlutusCoreReduction.tex

```diff
@@ -144,27 +143,27 @@
 
 \begin{figure*}[t]
     \judgmentdef{\(\cksteps{\sigma}{\sigma'}\)}{Machine state $\sigma$ transitions in one step to machine state $\sigma'$}
-    
+
     \begin{align*}
+        &\cksteps{\ckforward{s}{\con{cn}}}{\ckbackward{s}{\con{cn}}}\\
         &\cksteps{\ckforward{s}{\abs{\alpha}{K}{M}}}{\ckbackward{s}{\abs{\alpha}{K}{M}}}\\
```

Comment:
Replace `M` by `V` due to the value restriction?

Reply:
Done.